packit_service/worker/handlers/distgit.py

```diff
@@ -422,19 +443,35 @@ def pre_check(self) -> bool:
                 )
                 return False
 
-            # Packit should only build its own contributions
-            # We need to preserve the proven packager workflow - pushes not done by Packit
-            # should be built by those that do it, e.g. rebuilds in a side tag.
-            # prod/stg filtering happens in jobs.py -> packit_instances
-            if not (
-                self.data.event_dict["email"] == "hello@packit.dev"
-                and self.data.event_dict["name"] == "Packit"
-            ):
-                logger.info(
-                    f"Push event {self.data.identifier} ({self.data.event_dict['name']} "
-                    f"<{self.data.event_dict['email']}>) not done by us."
-                )
-                return False
+        if self.data.event_type == PushPagureEvent.__name__:
+            if self.data.event_dict["committer"] == "pagure":
+                closed_by = self.get_pr_closed_by()
```

Comment:
When I am looking at this now, I am not sure if this can't cause some unexpected builds. (Actually, it's not anyhow committed by that user..;)

@TomasTomecek @lbarcziova Do you think we can leave it like this or introduce the third option to make it really explicit?

Reply:
I don't really have a strong opinion, do you have any example of a use-case where it would be needed? :thinking: so that we don't have 3 configuration options if it is not really needed.

Reply:
The question is if we want to allow configuration for those who performed the merge at all. What about removing this check? So, in the case of PRs, we will check just an author, for pushes, will check committers.

I am really not sure. I am just worried that it might be confusing to have a config option for `committers` and use it for something else....

Reply:
Ok, I understand now. So I can remove the check for the person who merged the PR (and update the docs accordingly) and in case of someone in the future requests it, it shouldn't be a problem to implement the third option.